### ISSUE TITLE
[BUG] Error in a Doom module: modules/lang/common-lisp/config.el, (void-variable sly-contribs)

### ISSUE THREAD

**Issue by flyingns** [hlissner/doom-emacs#1772]
**Describe the issue**
Doom update to commit `d866ee3`, when emacs is started,
Prompt for this error.

**System information**
<details><pre>
mac mojave
emacs-plus 26.3
</pre></details>

**Comment by filalex77** [comment]
Can't reproduce on MacOS Mojave and Emacs 27 (master). How did you install Common Lisp?

**Comment by flyingns** [comment]
> Can't reproduce on MacOS Mojave and Emacs 27 (master). How did you install Common Lisp?

brew install sbcl.

**Comment by flyingns** [comment]
After many times `doom rc` & `doom refresh`, the error message has disappeared!

baffling...

**Comment by krathdog** [comment]
I got this same message. I just moved from straight to develop (to 42df2eaf8). I'm also on macos mojave, using 26.2, and also brew installed sbcl.

**Comment by krathdog** [comment]
I did a doom rebuild -f and reproduced this issue again.
I did a doom rc & doom refresh and it still happened. I did doom refresh -f and it went away.

**Comment by hlissner** [comment]
@krathdog What commit of Doom are you on? I pushed f03eb6731 ~14 hours ago, which should fix an issue with `doom rebuild` generating a broken autoloads file.

**Comment by krathdog** [comment]
I was on 42df2eaf8 which didn't have f03eb6731.

I did doom upgrade:

  Old revision: 42df2eaf8e (6 days ago)
  New revision: 86928e425b (2 hours ago)

Then doom rebuild -f. And hit this same issue again.

I did doom rc & doom refresh -f and it went away.

**Comment by macaroot** [comment]
On Arch, Emacs 26.3-1
Had this same issue. I found that by running doom rc I could always get the error, so I used that to check if it got fixed while I did things. Unfortunately I can't give detailed info, as I use doom as my desktop through exwm, so when doom is down I can't do much.

But doom rc stopped giving me the error after multiple runs of doom update/upgrade/refresh -f/rebuild -f. Then I relogged into doom and everything was fine.
